docs/full_backups.md

```diff
@@ -7,3 +7,10 @@ You can schedule full backups of your VMs, by exporting them to the local XOA fi
 :::tip
 Full backups are space consuming! But they allow a very simple restoration without anything to think of (the file will contain all the VM disks and information). To use less space and data transferred, take a look at the [delta backups](delta_backups.md) feature.
 :::
+## Backup without snapshot
+
+In some cases you will need to do a backup without doing a snapshot.  
+The most common use case for this is a large Vm on a small local SR where you just don't have the space to do a snapshot before backup .  
+So for that you need to open the advanced settings of your backup job, and check the offline backup check box.  
```

Comment:
"[...] backup check box:" (semicolon is inviting to read the image after)

Reply:
Colon "backup check box:" or semicolon "backup check box;"

Reply:
Sorry I meant colon `:`. Too many JS done in the past :p